packages/flutter_tools/lib/src/commands/build_appbundle.dart

```diff
@@ -35,11 +41,23 @@ class BuildAppBundleCommand extends BuildSubCommand {
     usesAnalyzeSizeFlag();
     addAndroidSpecificBuildOptions(hide: !verboseHelp);
     argParser.addMultiOption('target-platform',
-        splitCommas: true,
-        defaultsTo: <String>['android-arm', 'android-arm64', 'android-x64'],
-        allowed: <String>['android-arm', 'android-arm64', 'android-x64'],
-        help: 'The target platform for which the app is compiled.',
-      );
+      splitCommas: true,
+      defaultsTo: <String>['android-arm', 'android-arm64', 'android-x64'],
+      allowed: <String>['android-arm', 'android-arm64', 'android-x64'],
+      help: 'The target platform for which the app is compiled.',
+    );
+    argParser.addFlag('deferred-components',
+      negatable: true,
+      defaultsTo: true,
+      help: 'Disables building with deferred components.',
```

Comment:
Describe the implications of them being disabled. Does the app not work anymore or are everything bundled together? What are the performance characteristics?